css/style.css

```diff
@@ -52,18 +66,8 @@ main {
     justify-content: space-between;
 }
 
-#start {
-    padding: 40px 80px;
-    font-size: 2.5rem;
-}
-
-.instructions {
-    border: 1px solid #000000;
-    margin-top: 30px;
-}
-
-.instructions ol {
-    text-align: left;
+.chosen_number {
+    background-color: #505050;
```

Comment:
some opacity here would make it look a bit better but no big deal haha.

Reply:
Yeah we aren't going for style, can be dealt with later.